src/App.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from "styled-components";
+
+const Wrapper = styled.h1`
+  font-size: 1.5em;
+  text-align: center;
+  color: palevioletred;
+`;
+
+const App: React.FC = () => {
+  return (
```

Comment:
We have to get along about return should be mandatory 

`
const App: React.FC = () => {
    return <div>Starter</div>
}
`
or use direct return
`
const App: React.FC = () => (
    <div>Starter</div>
)
`

Reply:
That true, we need to check others request and make a decision.